test/main.ts

```diff
@@ -983,13 +1113,13 @@ suite('IDBPObjectStore', () => {
     const schemaDB = await openDBWithData();
     db = schemaDB as IDBPDatabase;
 
-    const store1 = schemaDB.transaction('object-store').store;
+    const store1 = schemaDB.transaction('object-store', 'versionchange').store;
```

Comment:
huh, does this test actually pass?

Reply:
Yes, I run `npm run build` locally, and no error popup.

Reply:
I ran the tests in the browser and they failed.

<img width="1092" alt="Screenshot 2020-09-11 at 14 16 45" src="https://user-images.githubusercontent.com/93594/92930101-8763cf80-f439-11ea-86db-22d02f50b113.png">

To run the tests:

```sh
npm run dev & npx serve
```

Then go to http://localhost:5000/build/test/

Reply:
Yeah, since we should only use the 'versionchange' transaction in upgrade method.